src/App.tsx

```diff
@@ -7,6 +7,7 @@ import TestBB from './pages/TestBB';
 import TestSS from './pages/TestSS';
 import TestVS from './pages/TestVS';
 import TestSM from './pages/TestSM';
+import TestMB from './pages/TestMB';
```

Comment:
Since you implemented the whole page it can be `import LoginPage` and update the file name as well

Reply:
Sounds good, and easy to ready. will make a change